tests/test_wiki_ES.py

```diff
@@ -0,0 +1,71 @@
+from apistar.test import TestClient
+from freezegun import freeze_time
+from app import app
+import pytest
+import os
+import re
+import json
+from .test_api import load_poi
+import responses
+
+@pytest.fixture(scope="session")
+def basket_ball_wiki_es(wiki_es_client, init_indices):
+    """
+    fill the elasticsearch WIKI_ES with a POI of basket ball
+    """
+    filepath = os.path.join(os.path.dirname(__file__), 'fixtures', 'basket_ball_wiki_es.json')
+    with open(filepath, "r") as f:
+        poi = json.load(f)
+        poi_id = poi['id']
+        wiki_es_client.index(index='wikidata_fr',
+                        body=poi,
+                        doc_type='wikipedia',
+                        id=poi_id,
+                        refresh=True)
+        return poi_id
+
+@pytest.fixture(scope="session")
+def basket_ball(es_client):
+    """
+    fill elasticsearch with a fake POI of basket ball
+    """
+    return load_poi('basket_ball.json', es_client)
+
+@freeze_time("2018-06-14 8:30:00", tz_offset=2)
+def test_basket_ball(basket_ball, basket_ball_wiki_es):
```

Comment:
nice test

it would be nice to add a bit more tests though:
* that if ES is not set we still call wikipedia api
* if ES has not the poi we do not call wikipedia
* if we don't have the lang we call wikipedia

Reply:
nice test ideas,
done